nirc_ehr/test/src/org.labkey.test/tests.nirc_ehr/NIRC_EHRTest.java

```diff
@@ -202,6 +214,25 @@ private void enableNotification(String notification)
         notificationAdminPage.enableRequestAdminAlerts(notification);
     }
 
+    private void populateEHRLookups() throws IOException, CommandException
```

Comment:
I'd rename this to populateLocations() to avoid confusion, since we are populatingEHRLookups in populateInitialData() which populates many ehr_lookup tables

Reply:
Done